tools/evmc/main.cpp

```diff
@@ -23,10 +43,10 @@ int main(int argc, const char** argv)
         *app.add_option("--vm", vm_config, "EVMC VM module")->envname("EVMC_VM");
 
     auto& run_cmd = *app.add_subcommand("run", "Execute EVM bytecode");
-    run_cmd.add_option("code", code_hex, "Hex-encoded bytecode")->required();
+    run_cmd.add_option("code", code_hex, "Bytecode")->required()->check(Hex);
```

Comment:
So this replaces `required`, does that mean HexValidator should fail if the input is shorter than 2 nibbles?

Reply:
This also raises the question, is there a CLI test for calling this without input bytecode?

Reply:
The "required" mean the argument is required. Values are always validated. Any order works.

Reply:
Oh, for some reason the split diff didn't show it to be already chained. So just ignore this question then.